homeassistant/components/fibaro/sensor.py

```diff
@@ -17,33 +22,32 @@
 
 from . import FIBARO_DEVICES, FibaroDevice
 
-SENSOR_TYPES = {
-    "com.fibaro.temperatureSensor": [
-        "Temperature",
-        None,
-        None,
-        SensorDeviceClass.TEMPERATURE,
-    ],
-    "com.fibaro.smokeSensor": [
-        "Smoke",
-        CONCENTRATION_PARTS_PER_MILLION,
-        "mdi:fire",
-        None,
-    ],
-    "CO2": [
-        "CO2",
-        CONCENTRATION_PARTS_PER_MILLION,
-        None,
-        None,
-        SensorDeviceClass.CO2,
```

Comment:
Looks like this PR is probably fixing a bug, too! (The "CO2" entry previously had 5 elements, whereas all the others have 4, which I think would have caused CO2 sensors to have no device class.)